mech_client/mech_marketplace_tool_management.py

```diff
@@ -7,8 +7,9 @@
 
 import requests
 from aea_ledger_ethereum import EthereumApi
+from web3.constants import ADDRESS_ZERO
 
-from mech_client.marketplace_interact import ADDRESS_ZERO, get_contract, get_mech_config
+from mech_client.interact import get_contract, get_mech_config
```

Comment:
Why just interact and not marketplace_interact ?

Reply:
Issues due to cyclic imports